README.md

```diff
@@ -1,22 +1,40 @@
 <br />
 
 <p align="center">
-    <img src="./.github/static/logo.svg" alt="Metrom logo" width="60%" />
+    <img src=".github/static/logo.svg" alt="Metrom logo" width="60%" />
 </p>
 
 <br />
 
 <p align="center">
-    Design your incentives to AMMplify liquidity.
+    MDesign your incentives to AMMplify liquidity
```

Comment:
Typo here

Reply:
Fixed in 198a2f7d31b8c4b0a2962a76f84a32308dcc34c0.